Source/ThirdParty/skia/CMakeLists.txt

```diff
@@ -0,0 +1,1019 @@
+set_property(DIRECTORY . PROPERTY FOLDER "skia")
+
+if (SKIA_ROOT_DIR OR NOT "$ENV{SKIA_ROOT_DIR}" STREQUAL "")
+    if (NOT SKIA_ROOT_DIR)
+        set(SKIA_ROOT_DIR "$ENV{SKIA_ROOT_DIR")
+    endif ()
+
+    find_path(SKIA_INCLUDE_DIR
+        NAMES include/private/base/SkFeatures.h
+        HINTS ${SKIA_INCLUDE_DIR} ${SKIA_ROOT_DIR}
+        NO_DEFAULT_PATH
+        REQUIRED
+    )
+    find_library(SKIA_LIBRARY
+        NAMES skia libskia
+        HINTS ${SKIA_LIBRARY_PATH} ${SKIA_ROOT_DIR}/out ${SKIA_ROOT_DIR}/out/Static
+        NO_DEFAULT_PATH
+        REQUIRED
+    )
+    find_path(SKIA_NINJA_RULES
+        NAMES skia.ninja
+        HINTS ${SKIA_LIBRARY_PATH} ${SKIA_ROOT_DIR}
+        PATH_SUFFIXES out/obj out/Static/obj
+        NO_DEFAULT_PATH
+        REQUIRED
+    )
```

Comment:
I don't think the finds are needed with the assumption that the tree is in source.

Reply:
Correct, let's see if we can remove all the support for external Skia builds.

Reply:
We would like to keep this for now, as discussed above.